packages/core/types/@chainlink/index.d.ts

```diff
@@ -58,8 +58,9 @@ declare module '@chainlink/types' {
   export type Callback = (statusCode: number, data?: any) => void
   export type AdapterHealthCheck = (callback: Callback) => any
 
-  import type { RequestConfig } from '@chainlink/external-adapter'
-  export type RequestConfig = RequestConfig
+  // TODO: fix circular dependency
```

Comment:
Here was our problem with that error on `yarn setup`. I think our solution is probably to combine `bootstrap` and `external-adapter`

Reply:
I agree, let's remove `external-adapter` package and merge it into `ea-bootstrap` 👍